src/api/app/views/webui/comment/_comment_field.html.haml

```diff
@@ -1,7 +1,7 @@
-= form_for(comment, action: 'create', method: :post, remote: true, html: { class: 'new-comment-form' }) do |f|
+= form_for(comment, method: form_method, remote: true, html: { class: "#{form_method}-comment-form" }) do |f|
   = hidden_field_tag :commentable_type, commentable.class.name
   = hidden_field_tag :commentable_id, commentable.id
   = f.hidden_field :parent_id, value: comment.parent_id
   ~ f.text_area :body, rows: '4', placeholder: 'Add a new comment (markdown markup supported)', required: true,
```

Comment:
We should have a more generic placeholder now that we will be using this partial for various actions (create and update). Something like:
```
Write your comment here... (Markdown markup is supported)
```